docs/src/rules/no-new.md

```diff
@@ -42,8 +42,6 @@ Examples of **correct** code for this rule:
 /*eslint no-new: "error"*/
 
 var thing = new Thing();
-
-Thing();
```

Comment:
Thanks for the PR @sir-kain, but I think this line was added intentionally to show that the rule does not report a problem if a constructor is called without `new`.

Reply:
Thank you. In this case, would it be better to provide 2 separate examples to avoid confusion? (As we already do with [lines-between-class-members](https://eslint.org/docs/latest/rules/lines-between-class-members#rule-details) rule)


For instance:

## Examples of **correct** code for this rule:

```js
/*eslint no-new: "error"*/

var thing = new Thing();
```

## Examples of additional **correct** code for this rule:

```js
/*eslint no-new: "error"*/

Thing();
```

Reply:
> Thank you. In this case, would it be better to provide 2 separate examples to avoid confusion? (As we already do with [lines-between-class-members](https://eslint.org/docs/latest/rules/lines-between-class-members#rule-details) rule)

My feeling is that `Thing();` alone does not make it immediately clear that a constructor is being invoked. This becomes evident when looking at the previous line. If the examples are split then there should be some additional explanation/context to clarify.

Reply:
In the incorrect example, the class is named `Thing`.

Reply:
This example aims to show that function calls are not reported even if the callee looks like a constructor, so `Foo();` would be fine.

Reply:
PR update.